docs/modules/ROOT/pages/getting-started/installation.adoc

```diff
@@ -0,0 +1,64 @@
+= Installation
+
+This document explains how to install and configure KubeArchive in your Kubernetes cluster.
+
+== Prerequisites
+
+* A PostgreSQL instance
+* CertManager is installed on the Kubernetes cluster (+v1.9.1)
+* Knative Eventing is installed on the Kubernetes cluster (+v1.15.0)
+
+== Prepare the Database
+
+. Create a user and a password for KubeArchive to use.
+. Create a database for the user to use.
+. As the user, execute the `ddl.sql` we provide in the database, located at:
++
+[source,bash,subs="attributes"]
+----
+ifeval::["{page-component-display-version}" == "main"]
+https://github.com/kubearchive/kubearchive/releases/latest/download/ddl.sql
+endif::[]
+ifeval::["{page-component-display-version}" != "main"]
+https://github.com/kubearchive/kubearchive/releases/download/{page-component-display-version}/ddl.sql
+endif::[]
+----
++
+This file creates a PostgreSQL schema, a table for KubeArchive and other stuff KubeArchive
+requires.
```

Comment:
Maybe we can also point the user to a way to apply directly through `psql` CLI

```suggestion
This file creates a PostgreSQL schema, a table for KubeArchive and other stuff KubeArchive
requires. It can be applied with `-f` option of `psql` CLI. See
link:https://www.postgresql.org/docs/current/app-psql.html#APP-PSQL-OPTION-FILE[docs].
```

Reply:
I changed the block slightly, now its `wget` + `psql` with example parameters.